cmd/subnetcmd/configure.go

```diff
@@ -0,0 +1,103 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package subnetcmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/spf13/cobra"
+)
+
+// avalanche subnet configure
+func newConfigureCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "configure [subnetName]",
+		Short:        "",
+		Long:         ``,
+		SilenceUsage: true,
+		RunE:         configure,
+		Args:         cobra.ExactArgs(1),
+	}
+	return cmd
+}
+
+func configure(cmd *cobra.Command, args []string) error {
+	chains, err := validateSubnetNameAndGetChains(args)
```

Comment:
We should validate the allowed characters in our subnet name and make sure none of them aren't illegal path characters like `/` or `?`.

Reply:
Not sure I understand. Aren't we doing this already in `checkInvalidSubnetNames` which is checked in `validateSubnetNameAndGetChains`?